Sources/SwiftDocC/Indexing/Navigator/NavigatorIndex.swift

```diff
@@ -63,7 +63,7 @@ public class NavigatorIndex {
     public enum Error: Swift.Error, DescribedError {
         
         /// Missing bundle identifier.
-        case missingBundleIndentifier
+        case missingBundleIdentifier
```

Comment:
Since this is a public symbol, renaming it is technically a source breaking change.

```suggestion
        case missingBundleIdentifier
        
        @available(*, deprecated, renamed: "missingBundleIdentifier")
        case missingBundleIndentifier
```